skorch/callbacks/logging.py

```diff
@@ -875,3 +875,70 @@ def on_epoch_end(self, net, **kwargs):
 
         for key in filter_log_keys(epoch_logs.keys(), self.keys_ignored_):
             self.experiment.log_scalar(key + self.epoch_suffix_, epoch_logs[key], epoch)
+
+
+class MLflow(Callback):
+    """
+    >>> import mlflow
+    >>> net = NeuralNetClassifier(net, callbacks=[MLflow])
+    >>> with mlflow.start_run():
+    ...     net.fit(X, y)
+    """
+    def __init__(
+        self,
+        log_on_batch_end=False,
+        log_on_epoch_end=True,
+        keys_ignored=[],
+        create_artifact=True,
+        logged_parameters=["lr", "batch_size"],
```

Comment:
The other loggers let the user register the parameters manualy but I thought it could have it's place here

Reply:
Could you elaborate?